dart/lib/src/sentry_client.dart

```diff
@@ -60,37 +56,39 @@ class SentryClient {
       return _sentryId;
     }
 
-    event = _prepareEvent(event, stackTrace: stackTrace);
+    SentryEvent? preparedEvent = _prepareEvent(event, stackTrace: stackTrace);
```

Comment:
probably the same question as above, copying the reference instead of using it directly

Reply:
Also same answer as above. The parameter is non-nullable, but it can be null after processing.